services/121-service/src/connection/dto/update-attribute.dto.ts

```diff
@@ -0,0 +1,22 @@
+import { ApiModelProperty } from '@nestjs/swagger';
+import { Length, IsString, IsIn } from 'class-validator';
+
+export enum AttributesEnum {
+  paymentAmountMultiplier = 'paymentAmountMultiplier',
+}
+const attributesArray = Object.values(AttributesEnum).map(item => String(item));
+
+export class UpdateAttributeDto {
+  @ApiModelProperty({ example: '910c50be-f131-4b53-b06b-6506a40a2734' })
+  @Length(29, 36)
+  public readonly referenceId: string;
+  @ApiModelProperty({
+    enum: attributesArray,
+    example: attributesArray.join(' | '),
+  })
+  @IsIn(attributesArray)
+  public readonly attribute: AttributesEnum;
+  @ApiModelProperty({ example: 'new value' })
+  @IsString()
+  public readonly value: string;
```

Comment:
Not all values need to be `string`s right?

Reply:
changed